pkg/component/component.go

```diff
@@ -94,9 +94,22 @@ func CreateFromPath(client *occlient.Client, name string, ctype string, path str
 	return nil
 }
 
+// CreateFromTemplate create new component from serviceCatalog
+// we are using template service broker only
+func CreateFromTemplate(client *occlient.Client, componentName string, componentType string, applicationName string) error {
```

Comment:
This is confusing, commend and function name says that you are creating it from a template, but in fact you just ServiceInstatnce, that is not limited to just template.